packages/components/src/ActionList/ActionListItem.tsx

```diff
@@ -48,9 +48,17 @@ const ActionListItemInternal: FC<ActionListItemProps> = ({
   onClick,
 }) => {
   const actionListItemRef = useRef<HTMLDivElement>(null)
-  const { canSelect, itemsSelected, onSelect, onClickRowSelect } = useContext(
-    ActionListContext
-  )
+  const {
+    addItemToAllItems,
+    canSelect,
+    itemsSelected,
+    onSelect,
+    onClickRowSelect,
+  } = useContext(ActionListContext)
+
+  useEffect(() => {
+    !disabled && addItemToAllItems(id)
+  }, [addItemToAllItems, disabled, id])
```

Comment:
Add all items regardless of their disabled state - I think it feels funny otherwise.

```suggestion
  useEffect(() => addItemToAllItems(id), [addItemToAllItems, id])
```

I wonder if perhaps the final fix here (in a future PR) is an option to omit the checkbox for those items that are disabled?

Reply:
Resolved in https://github.com/looker-open-source/components/pull/837/commits/0c8b5a8b1cfd4c930a5891ba1a42a93a77412fa7!

Reply:
And I actually like the look of omitted checkboxes for disabled rows. I added it in since it was a minor change, and thought it was worth including in this PR. (change included in commit bbd4a3010).

![Screen Shot 2020-04-29 at 4 36 40 PM](https://user-images.githubusercontent.com/16812885/80657218-dcc5b400-8a37-11ea-81a9-dc920431ddd9.png)

Reply:
Let me know if you'd rather keep this change out of the current PR!

Reply:
I want to run that by design before we make that change so let's pull it out for now :)

Reply:
🖖